sherpa-onnx/csrc/session.cc

```diff
@@ -64,26 +66,51 @@ static Ort::SessionOptions GetSessionOptionsImpl(int32_t num_threads,
       break;
     }
     case Provider::kTRT: {
+      if (provider_config == nullptr) {
+        SHERPA_ONNX_LOGE("Tensorrt support for Online models ony,"
+                         "Must be extended for offline and others");
+        exit(1);
+      }
+      auto trt_config = provider_config->trt_config;
       struct TrtPairs {
         const char *op_keys;
         const char *op_values;
       };
 
+      auto device_id = std::to_string(provider_config->device);
+      auto trt_max_workspace_size =
+          std::to_string(trt_config.trt_max_workspace_size);
```

Comment:
Since you are converting integers to strings, please replace all
`uint32_t` with `int64_t`. Remember unsigned integers are error-prone. Please avoid using them if possible.

Reply:
I will make all the uint32 to int32, but for int64_t, SHERPA_ONNX_LOGE() format will be like below

#include <inttypes.h>
int64_t t;
printf("%" PRId64 "\n", t);

will raise the patch in sometime